examples/build.gradle

```diff
@@ -1,5 +1,5 @@
-project.ext.sdkVersion = 23
-project.ext.buildTools = '23.0.0'
+project.ext.sdkVersion = 24
```

Comment:
The SDK version should not have any effect on what is actual running on the device. This is just used to compile everything with.